consensus/merge/src/test/java/org/hyperledger/besu/consensus/merge/blockcreation/MergeCoordinatorTest.java

```diff
@@ -213,7 +213,8 @@ public void coinbaseShouldMatchSuggestedFeeRecipient() {
             genesisState.getBlock().getHeader(),
             System.currentTimeMillis() / 1000,
             Bytes32.ZERO,
-            suggestedFeeRecipient);
+            suggestedFeeRecipient,
+            Optional.empty());
```

Comment:
nit: could extract a variable to make this clearer it's emptyWithdrawals

Reply:
done